### ISSUE TITLE
github action: file an issue when upstream makes a tagged release

### ISSUE THREAD

**Issue by meowsbits** [etclabscore/core-geth#89]
We'll want to merge it.  Would be nice to have an automatic notification and ticket for it.

**Comment by ziogaschr** [comment]
Just got this [wei/pull](https://github.com/wei/pull) GitHub Action from an email I receive with GitHub repos.
What do you think?

**Comment by meowsbits** [comment]
Interesting... nice find. For starters I'll try it out on meowsbits/core-geth just to see what it's like.. 

I like that:
- it's an automatic way to put work in front of us
- it will make merges (and their conflict resolutions) transparent and reviewable

**Comment by ziogaschr** [comment]
Yes, I remembered that I have a request for such a package somewhere and searched at our issues. Seems to be what we need.

I would try to hook into the release branches of go-ethereum if it's not possible to use tags.

**Comment by meowsbits** [comment]
The other _kind of crazy_ idea is to do 1:1 PRs... :grimacing: :neckbeard:

**Comment by ziogaschr** [comment]
:) My only worry would be that we will have to handle a tone of PRs. Except if we make a new branch (not master) where it will merge the PRs automatically, except if conflicts exist
